### PR TITLE
Rename tools directory to utils (grass8 branch)

### PR DESCRIPTION
**Pull request by neteler**
Apply directory naming from grass-code also here (https://github.com/OSGeo/grass/pull/1348):

- utils is more common for helper scripts then tools which is too close to modules.

### CODE REVIEW

File: utils/addons/grass-addons-index.sh [hunk 1]
```diff
@@ -92,8 +92,8 @@ generate () {
 <body bgcolor=\"#FFFFFF\">
 <h2>GRASS GIS ${major} Addons Manual pages</h2>
 
-<!-- Generated from: /home/martinl/src/grass_addons/tools/addons/ -->
-<!--       See also: https://github.com/OSGeo/grass-addons/blob/master/tools/addons/README.md -->
+<!-- Generated from: /home/martinl/src/grass_addons/utils/addons/ -->
+<!--       See also: https://github.com/OSGeo/grass-addons/blob/master/utils/addons/README.md -->
```
Comment (wenzeslaus) [inline utils/addons/grass-addons-index.sh:96]: This and other URLs are now quite outdated, but perhaps they don't need to be URLs at all when they point to files in the same repo.

**Review by wenzeslaus — APPROVED** [review]
Looks straightforward.

File: utils/addons/grass-addons-index.sh (hunk 1 above)
Comment (neteler) [reply utils/addons/grass-addons-index.sh:96]: The link is actual once this PR is merged.

Comment (neteler) [reply utils/addons/grass-addons-index.sh:96]: Done in 51b3a84a6612528b879b1619f947afc773f3e79f

**Pull request merged**